Pipeline/PackageBuildOutputs/README.md

```diff
@@ -45,6 +45,44 @@ This section provides a more detailed explanation of how the PackageBuildOutputs
 Notes: 
 * The script doesn't manage the deletions of artifacts. Although they are reported in the DBB Build Reports, deletions are not handled by this script.
 
+## Software-Bill-Of-Material (SBOM) generation
+
```

Comment:
The section `Package Build Outputs Process - High-level Processing Flow` is missing the SBOM generation step.

And, I feel this section is better suited below the CLI summary as details of the SBOM generation feature, while the overview could have a link to the section.

Reply:
Done